src/config/locales.md

```diff
@@ -9,7 +9,7 @@ locale -a
 ## Enabling locales
 
 To enable the locale `xxxx`, un-comment or add the relevant line in
-`/etc/default/libc-locale.conf` and run
+`/etc/default/libc-locale` and run
```

Comment:
this is embarrasing. but isn't it `libc-locales`?

Also, this should really be in a separate PR.